app/javascript/admin/controllers/config_controller.js

```diff
@@ -245,4 +259,64 @@ export default class ConfigController extends Controller {
     }
   }
   // AUTH PROVIDERS FUNCTIONS END
+
+  enabledProvidersWithMissingKeys() {
+    const providersWithMissingKeys = [];
+    document
+      .querySelectorAll('[data-enable-auth="true"]')
+      .forEach((provider) => {
+        const providerName = provider.dataset.authProviderEnable;
```

Comment:
This is a bit confusing to me...why are we setting a variable to `providerName` to something called `authProviderEnable`? 🤔

Reply:
I guess it is a little confusing 😅  The data attribute `auth-provider-enable` is named such because when I first used it, it was to identify which providers are enabled when their "Enable" button is clicked.

Reply:
@nickytonline @vaidehijoshi Hey y'all, thanks for your input above. I understand what you are suggesting; however, I am pretty sure that the way I have things named currently are so for good reasons.
The development of `/admin/config` has been through several design and UX-flow iterations, with each one requiring the changes I have made thus far.
I want to create some sort of walkthrough explaining the pieces of what I have built, and why I have named things the way I have. I cannot do that right now because it requires too much of a mental switch.
I don't mean to imply that these data attributes cannot be named better, but I'd like to take a proper step back first and make the changes in a context of understanding.
I hope you do not mind 🙏🏾